.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @hashicorp/terraform-ecosystem-kubernetes
+* @snyk/cloud-platforms
```

Comment:
I think we need a few more details about this change. 

Is this a hostile takeover? :)